integration/networking/windows_drivers_test.go

```diff
@@ -163,7 +166,6 @@ func TestWindowsNetworkDNSResolution(t *testing.T) {
 			)
 			defer c.ContainerRemove(ctx, id, client.ContainerRemoveOptions{Force: true})
 
-			// Allow time for DNS registration
 			time.Sleep(2 * time.Second)
```

Comment:
This comment may be good to keep; a `time.Sleep` in tests usually is a hack/workaround, and it's good to describe what the motivation was to add it, so help future visitors to understand / decide whether it should (still) be there.